File: mssql/php/read/mssql_read.php
#! /usr/bin/php
<?php
/*
	mssql_read.php

					Nov/18/2015

*/
// --------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);

include "text_manipulate.php";
// --------------------------------------------------------------------
print "*** 開始 ***\n";
ini_set('display_errors', 1);

$server = 'host_mssql\EG';
$user = 'sa';
$password = 'Tiger123';

$link = mssql_connect ($server,$user,$password);

if (!$link)
	{
	die('\n\nSomething went wrong while connecting to MSSQL');
	}
else
	{
	$selected = mssql_select_db ("city")
		or die ("Couldn’t open database databasename\n");

	echo "connected to databasename\n";
    
	$result = mssql_query ("select * from cities");

	$dict_aa = array ();

	while($row = mssql_fetch_array($result))
		{
		$name = $row["name"];
		$key=$row["id"];
		$dict_unit = array ();
		$dict_unit['name'] = $row["name"];
		$dict_unit['population'] = $row["population"];
		$dict_unit['date_mod'] = $row["date_mod"];
		$dict_aa[$key]= $dict_unit;
		}
	}

mssql_close();

dict_display_proc ($dict_aa);

print "*** 終了 ***\n";
// --------------------------------------------------------------------
?>

File: mssql/python/read/mssql_read.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	mssql_read.py
#
#						Aug/30/2017
#
#
# ----------------------------------------------------------------
import	sys
import	pymssql
#
sys.path.append ('/var/www/data_base/common/python_common')
from sql_manipulate import sql_to_dict_proc
from text_manipulate import dict_display_proc
# ----------------------------------------------------------------
#
#
print ("*** 開始 ***")
#
server_mssql = 'localhost'
user_mssql = 'sa'
password_mssql = 'Sql12345'
#
try:
	conn = pymssql.connect \
		(host=server_mssql,user=user_mssql,password=password_mssql,database='city')
	cursor = conn.cursor(as_dict=True)
#
	dict_aa=sql_to_dict_proc (cursor)
#
	cursor.close ()
	conn.close ()
#
	dict_display_proc (dict_aa)
#
except Exception as ee:
		sys.stderr.write ("*** error *** mssql_read.py ***\n")
		sys.stderr.write (str (ee))
#
print ("*** 終了 ***")
#
# ----------------------------------------------------------------
